kyo-core/shared/src/main/scala/kyo/core.scala

```diff
@@ -27,13 +27,13 @@ object core:
                 def apply(v: T, s: Safepoint[E], l: State) =
                     v
 
-        inline def suspend[T, U, S](inline cmd: e.Command[T], inline f: T => U < S)(
+        inline def suspend[T, U, S](inline cmd: e.Command[T], inline f: T => U < (E & S))(
```

Comment:
Just makes the API more flexible by allowing the `f` continuation to also perform the effect.